docs/index.md

```diff
@@ -565,7 +568,7 @@ Flags:
       --skip-deps                         skip running "helm repo update" and "helm dependency build"
       --state-values-file stringArray     specify state values in a YAML file. Used to override .Values within the helmfile template (not values template).
       --state-values-set stringArray      set state values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2). Used to override .Values within the helmfile template (not values template).
-      --strip-args-values-on-exit-error   On exit error, strip the values of the args
+      --strip-args-values-on-exit-error   Strip the potential secret values of the helm command args contained in a helmfile error message (default true)
```

Comment:
please update the description in the go file.

Reply:
The output above was generated by executing `helmfile --help`